var/spack/repos/builtin/packages/py-jupyterhub/package.py

```diff
@@ -10,22 +10,21 @@ class PyJupyterhub(PythonPackage):
     """Multi-user server for Jupyter notebooks."""
 
     homepage = "https://pypi.org/project/jupyterhub"
-    url      = "https://pypi.io/packages/source/j/jupyterhub/jupyterhub-0.9.4.tar.gz"
+    url      = "https://pypi.io/packages/source/j/jupyterhub/jupyterhub-1.0.0.tar.gz"
 
-    version('0.9.4',    sha256='86b1cce446d4e8347e26913878858fc8964d103fde19b606fe37ccc5188d629d')
```

Comment:
Can you keep the old release?

Reply:
I will need to check if the old version required node-js. If not I'll put it back.

Reply:
The old version depended on a few things that spack doesn't provide, so it will take some more new packages to actually make it work (the new version changed dependencies and should be good to go). Do you think I should add the dependencies to make the previous version work or leave it out?

Reply:
I usually get yelled at when I remove old versions. If you're willing to get yelled at and add the old version back in if someone requests it, then you can remove it for now.